deployment/provision

```diff
@@ -72,24 +72,15 @@ git clone https://github.com/andrewn/neue-radio rde
 cd rde/shared/websocket
 npm install --production
 cd ../..
-cd services/manager
-npm install --unsafe-perm --production
-cd ..
-cd physical
-JOBS=MAX npm install --unsafe-perm --production # use all cores
-cd ..
-cd rfid
-JOBS=MAX npm install --unsafe-perm --production # use all cores
-cd ..
-cd debug
-JOBS=MAX npm install --unsafe-perm --production # use all cores
-cd ..
-cd setup
-JOBS=MAX npm install --unsafe-perm --production # use all cores
-cd ..
-cd speech
-JOBS=MAX npm install --unsafe-perm --production # use all cores
-cd ../..
+
+for f in apps/* services/*; do
+   if [ -d ${f} ]; then
+      cd $f
+      npm install --unsafe-perm --production
```

Comment:
I just looked up [`JOBS=MAX`](https://gist.github.com/ralphtheninja/f7c45bdee00784b41fed) and it's used to spread the compilation of native deps across multiple cores. This is quite useful on the Pi so maybe we should add it back in?